firstpartydata/first_party_data_test.go

```diff
@@ -743,6 +744,15 @@ type fpdFile struct {
 	ValidationErrors   []*errortypes.BadInput                             `json:"validationErrors,omitempty"`
 }
 
+type fpdFileForResolveFPD struct {
+	InputRequestData     json.RawMessage                                `json:"inputRequestData,omitempty"`
+	OutputRequestData    map[openrtb_ext.BidderName]openrtb2.BidRequest `json:"outputRequestData,omitempty"`
+	BiddersWithGlobalFPD []string                                       `json:"biddersWithGlobalFPD,omitempty"`
+	BidderConfigFPD      map[openrtb_ext.BidderName]*openrtb_ext.ORTB2  `json:"bidderConfigFPD,omitempty"`
+	GlobalFPD            map[string]json.RawMessage                     `json:"globalFPD,omitempty"`
+	ValidationErrors     []*errortypes.BadInput                         `json:"validationErrors,omitempty"`
+}
```

Comment:
Needed a different output format. To avoid more json unmashal code, i created a separate structure and changed the load mechanism to use generics.